### PR TITLE
chore: refine README.kr.md

### LINKED ISSUE #3889: 📎 Refine Korean README.md
### Description

Thanks to @Procrustes5’s excellent work in translating the README into Korean(#3580), but there are a few parts that might sound awkward to native speakers.

I'm planning to refine the Korean version of the README. May I go ahead?

- In Korean, we use `,`(comma) instead of `、`(japanese comma).
- The remaining links redirect to Japanese pages. (Many Koreans are more familiar with English than Japanese.)

### PR DESCRIPTION
**Pull request by StationSoen**
## Summary
Refine Korean README Translation.

- Update some sections that differ from the README.md.
- Refine some slightly awkward sentences in the Korean README.md.
- Update the Japanese link to the default link.

## Tickets
- closes #3889

## Test Plan
- I’ll suggest a native Korean speaker as a reviewer, if that’s okay with you.

### CODE REVIEW

**Comment by Conaclos** [comment]
> I’ll suggest a native Korean speaker as a reviewer, if that’s okay with you.

Yeah. Please :)

File: packages/@biomejs/biome/README.kr.md [hunk 1]
```diff
@@ -37,13 +40,13 @@
...
 
-**Biome** 은 _JavaScript_, _TypeScript_, _JSX_을 위한 **[고성능 Linter](https://github.com/biomejs/biome/tree/main/benchmark#linting)** 로、ESLint, typescript-eslint, [등의 리소스](https://github.com/biomejs/biome/discussions/3)에서 **[270개 이상의 룰](https://biomejs.dev/linter/rules/)** 을 제공하고 있습니다. Biome 은 **상세하며 문맥에 맞는 결과를 출력**하기 위해, 코드를 개선하고, 더 좋은 프로그래머가 되기 위한 도움을 드립니다!
+**Biome**은 _JavaScript_, _TypeScript_, _JSX_, _CSS_ 그리고 _GraphQL_ 을 위한 **[고성능 Linter](https://github.com/biomejs/biome/tree/main/benchmark#linting)** 로, ESLint, typescript-eslint, [등의 리소스](https://github.com/biomejs/biome/discussions/3)에서 **[270개 이상의 룰](https://biomejs.dev/linter/rules/)** 을 제공하고 있습니다. Biome은 코드를 개선하고, 더 좋은 프로그래머가 되기 위한 **상세하며 문맥에 맞는 결과를 출력**합니다!
 
-**Biome** 은 처음부터 [**에디터 내에서 인터렉티브하게**](https://biomejs.dev/ja/guides/integrate-in-editor/) 사용하도록 설계되어 있습니다.
+**Biome**은 [**에디터 내에서 인터렉티브하게**](https://biomejs.dev/guides/integrate-in-editor/) 사용하도록 설계되어 있습니다.
 여러분이 코드를 작성할 때, 형식이 잘못된 코드에 format, lint 를 적용합니다.
```
Comment (quiple) [inline packages/@biomejs/biome/README.kr.md:50]:
```suggestion
```

File: packages/@biomejs/biome/README.kr.md [hunk 2]
```diff
@@ -61,44 +64,44 @@ npx @biomejs/biome format --write ./src
...
+더 많은 정보는 [프로젝트 철학][biome-philosophy]에서 확인할 수 있습니다.
 
-**Biome** 은 [MIT 라이센스](https://github.com/biomejs/biome/tree/main/LICENSE-MIT) 혹은 [Apache 2.0 라이센스](https://github.com/biomejs/biome/tree/main/LICENSE-APACHE)로, [기여자 행동 규범](https://github.com/biomejs/biome/tree/main/CODE_OF_CONDUCT.md)에 따라 관리되고 있습니다.
+**Biome**은 [MIT 라이센스](https://github.com/biomejs/biome/tree/main/LICENSE-MIT) 혹은 [Apache 2.0 라이센스](https://github.com/biomejs/biome/tree/main/LICENSE-APACHE)로, [기여자 행동 규범](https://github.com/biomejs/biome/tree/main/CODE_OF_CONDUCT.md)에 따라 관리되고 있습니다.
 
 ## 펀딩
 
 다양한 방법으로 프로젝트를 지원할 수 있습니다.
 
 ### 프로젝트 스폰서와 펀딩
```
Comment (quiple) [inline packages/@biomejs/biome/README.kr.md:100]:
```suggestion
### 프로젝트 스폰서십 및 펀딩
```

File: packages/@biomejs/biome/README.kr.md [hunk 3]
```diff
@@ -61,44 +64,44 @@ npx @biomejs/biome format --write ./src
...
 
-Biome 을 설치하지 않고 사용해보고 싶다면, WebAssembly 에 컴파일된 [온라인 플레이그라운드](https://biomejs.dev/playground/)을 이용해주세요.
+Biome을 설치하지 않고 사용해보고 싶다면, WebAssembly로 컴파일된 [온라인 플레이그라운드](https://biomejs.dev/playground/)을 이용해주세요.
 
 ## 문서
 
-Biome 에 대해 알아보기 위해 [홈페이지][biomejs]를 체크하거나, Biome 을 사용하기 위해 [시작하기][getting-started]을 확인하세요!
+Biome 에 대해 알아보기 위해 [홈페이지][biomejs]를 체크하거나, Biome을 사용하기 위해 [시작하기][getting-started]을 확인하세요!
 
 ## Biome 를 자세히 알아보기
```
Comment (quiple) [inline packages/@biomejs/biome/README.kr.md:80]:
```suggestion
## Biome에 대해 자세히 알아보기
```

**Review by quiple — CHANGES_REQUESTED** [review]
원본 영어 문서에 맞춰서 업데이트 및 번역을 개선하고 맞춤법을 수정했습니다.

**Comment by ematipico** [comment]
@StationSoen can you apply the suggestions?

**Comment by StationSoen** [comment]
@ematipico Sorry for the delay. I’ve applied all the feedback since it was very helpful.

**Pull request merged**